src/molecule/api.py

```diff
@@ -51,14 +51,14 @@ def drivers(config=None) -> UserListMap:
     pm = pluggy.PluginManager("molecule.driver")
     try:
         pm.load_setuptools_entrypoints("molecule.driver")
-    except Exception:
+    except (Exception, SystemExit):
```

Comment:
@ssbarnea this doesn't catch *everything*, though. Is it intentional that you didn't use `BaseException`?

Reply:
I wasn't sure what kind of other exceptions would Base Exception catch so I went for that approach. Shortly I don't know if current code is better than just BaseException or not.

Reply:
You could catch all the exceptions excluding `KeyboardInterrupt`, I think.

Reply:
The whole hierarchy is documented here: https://docs.python.org/3/library/exceptions.html#exception-hierarchy

Reply:
OTOH Molecule could also define own exceptions for communicating misconfiguration clearly. In this case, you'd need something like
```python
try:
    ...
except KeyboardInterrupt:
    raise
except MoleculeMisconfiguredException:
    report_exc()
except BaseException:
    process_exc()
```